CHANGELOG.md

```diff
@@ -9,17 +9,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- The C++ driver container image is now available on [Docker hub](https://hub.docker.com/r/mongodb/mongo-cxx-driver).
 - Add CMake option `MONGOCXX_OVERRIDE_DEFAULT_INSTALL_PREFIX` (default is `TRUE`
```

Comment:
Suggest to add its purpose. Something like:

Add CMake option `MONGOCXX_OVERRIDE_DEFAULT_INSTALL_PREFIX` (default is `TRUE` for backwards-compatibility). If enabled, `mongocxx` will set a default `CMAKE_INSTALL_PREFIX` if one is not already defined.

Reply:
I like that idea. Updated.